components/binding-usage-controller/docs/usage-kind.md

```diff
@@ -0,0 +1,33 @@
+# UsageKind
+## Overview
+
+UsageKind allows dynamic, in the runtime, Binding Usage Controller configuration. It allows you to define what resources can be bound with ServiceBinding and how to bind them.
+
+## UsageKind structure
+
+From the technical point of view the binding process modifies target resource by adding labels to the Pod. The UsageKind specifies the field which contains labels added to the Pod and the resource type (group, kind and version). 
```

Comment:
``From the technical point of view, the binding process modifies the target resource by adding labels to the Pod. The UsageKind specifies the field which contains labels added to the Pod, and the resource type defined by the **group**, **kind**, and **version** fields. ``